app/src/main/java/com/x8bit/bitwarden/ui/platform/components/appbar/action/BitwardenOverflowActionItem.kt

```diff
@@ -65,23 +65,21 @@ fun BitwardenOverflowActionItem(
             containerColor = BitwardenTheme.colorScheme.background.primary,
             modifier = Modifier
                 .semantics { testTagsAsResourceId = true }
+                .testTag("FloatingOptionsContent")
                 .widthIn(
                     min = 112.dp,
                     max = 280.dp,
                 ),
             content = {
                 menuItemDataList.forEach { dropdownMenuItemData ->
                     DropdownMenuItem(
-                        modifier = Modifier.semantics {
-                            dropdownMenuItemData.testTag?.let {
```

Comment:
Please remove the `testTag` property from the `OverflowMenuItemData` if it is no longer is needed

Reply:
got it!